components/contract-components/tables/deployed-contracts.tsx

```diff
@@ -393,7 +395,25 @@ export const ContractTable: ComponentWithChildren<ContractTableProps> = ({
         Header: "Contract Address",
         accessor: (row) => row.address,
         Cell: (cell: any) => {
-          return <AddressCopyButton address={cell.row.original.address} />;
+          const data =
+            configuredChains[cell.row.original.chainId] ||
+            chainIdToChainRecord[cell.row.original.chainId];
+          const blockExplorerUrl = data?.explorers?.[0]?.url;
+
+          return (
+            <Flex align="center" gap={2}>
+              <AddressCopyButton address={cell.row.original.address} />
+              {blockExplorerUrl && (
+                <Link
+                  href={`${blockExplorerUrl}/address/${cell.row.original.address}`}
```

Comment:
is this guaranteed to be the correct URL scheme for all variants of block explorers?

Reply:
Agree. I added new changes to filter block explorer URL that follows the standard of [EIP3091](https://eips.ethereum.org/EIPS/eip-3091)

Reply:
> is this guaranteed to be the correct URL scheme for all variants of block explorers?

Agree. I added new changes to filter block explorer URL that follows the standard of [EIP3091](https://eips.ethereum.org/EIPS/eip-3091)